client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/RequestStats.tsx

```diff
@@ -1,49 +1,32 @@
 import React from 'react';
-import { Box } from '@openmsupply-client/common';
+import { Box, Paper } from '@openmsupply-client/common';
 import { DraftRequestLine } from '../hooks';
-import { StockDistribution } from './StockDistribution';
 import { ConsumptionHistory } from './ConsumptionHistory';
-import { StockEvolution } from './StockEvolution';
 
 export interface RequestStatsProps {
   draft?: DraftRequestLine | null;
 }
 
 export const RequestStats = ({ draft }: RequestStatsProps) => {
   return (
-    <Box
+    <Paper
       sx={{
+        mt: 2,
+        p: 2,
         minHeight: 200,
         maxHeight: 500,
-        width: 500,
+        width: '100%',
+        overflow: 'auto',
       }}
     >
-      {!!draft && (
-        <StockDistribution
```

Comment:
Didn't delete component for now, `StockDistribution` seems to be a well made/though out component. Will delete if deemed not needed

Reply:
Hm yeah, I doubt we were wanting to drop either of these charts... @roxy-dao @mark-prins could you confirm?

Reply:
Yes we still want to show the charts~ 

![Screenshot 2025-05-02 at 2 52 00 PM](https://github.com/user-attachments/assets/f570ec22-56f1-4ab9-bf1a-0d5ce64d0213)